Themes/default/MoveTopic.template.php

```diff
@@ -99,8 +99,18 @@ function template_redirect_options($type)
 							<dt>
 								', $txt[$type . '_why'], '
 							</dt>
-							<dd>
-								<textarea name="reason">', $txt[$type . 'topic_default'], '</textarea>
+							<dd>';
+	if ($type == 'move')
+	{
+		echo '
+								<textarea name="reason">', sprintf($txt[$type . 'topic_default'], $txt['movetopic_auto_board'], $txt['movetopic_auto_topic']), '</textarea>';
+	}
+	elseif ($type == 'merge')
+	{
+		echo '
+								<textarea name="reason">', sprintf($txt[$type . 'topic_default'], $txt['movetopic_auto_topic']), '</textarea>';
+	}
+	echo '
```

Comment:
Since you specified in the language strings the numerical indexes (%1, %2), you can just use one call here.  The other replacement will be ignored.